src/test/java/ch/jalu/configme/configurationdata/CommentsConfigurationTest.java

```diff
@@ -11,12 +11,31 @@
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 /**
  * Test for {@link CommentsConfiguration}.
  */
 class CommentsConfigurationTest {
 
+    @Test
+    void shouldThrowForExistingPath() {
+        // given
+        final String path = "config.me"; 
+        CommentsConfiguration conf = new CommentsConfiguration();
+        conf.setComment(path, "Old", "Comments", "1", "2", "3");
+        
+        // when
+        IllegalStateException ex = assertThrows(IllegalStateException.class, () -> conf.setComment(path, "New Comment"));
+
+        // then
+        assertThat(ex.getMessage(), equalTo("Comment lines already exists for the path " + path));
```

Comment:
Other minor thing: I would hardcode the path here so that we fully see what the exception message is:
```suggestion
        assertThat(ex.getMessage(), equalTo("Comment lines already exists for the path config.me"));
```

I know it's trivial to understand here, but keeping the full exception message as a plain string has helped me in the past to find things that were weirdly displayed that would have gone unnoticed otherwise. It also helps me to see the message in a plain string to think about whether the exception message I'm writing makes sense or not